src/CharacterState.test.js

```diff
@@ -7,18 +7,26 @@ import * as C2lcMath from './C2lcMath';
 //       with Flow than casting the expect() result to 'any'.
 
 expect.extend({
-    toHaveCharacterState(received, xPos, yPos, directionDegrees) {
+    toHaveCharacterState(received, xPos, yPos, directionDegrees, path) {
+        const isEqualPath = received.path[0] != null && path[0] != null ?
```

Comment:
Jest provides a utility function that does a deep equality check:

https://jestjs.io/docs/en/expect#thisequalsa-b

So, we can change the path test to:

    this.equals(received.path, path)

This will make the code simpler and also enable us to test paths with more than one segment.

Reply:
Sorry. Using a general deep equality check isn't going to work of course, as we need to use approximate equality testing on the coordinates. I think we should write a function for comparing paths.